### ISSUE TITLE
Is it possible to backup/export DSK codes?

### ISSUE THREAD

**Issue by schmitzkr** [zwave-js/zwave-js-ui#3296]
Hi there, in the past year I have started using zwave with zwave-js-ui on home assistant and now have several devices.  I also have a box of scraps of paper and cardboard that have all the DSK codes for each of my devices.
Is there a way to export this from zwave-js-ui ? (or elsewhere?)

Thanks!

**Comment by schmitzkr** [comment]
Thank you, my home assistant instance is running in a container, I have no idea where these cache files are.

I added this snippet and scheduled it, I think its working but I am not able to decipher the log, looking at the snippet I expect to see a series of lines (one for each node) that looks like this 001:85392-40675-66566-65666-45987-32333-32323-98989
where 001 is the nodeID and the string of numbers and dashes is the dsk.
Is this correct?

**Comment by schmitzkr** [comment]
Thanks, I was able to get the DSK from the debug log now.
Scheduling as a cron job in the general settings (this was the first place I found that I could add a snippet)
Thanks again!

**Comment by robertsLando** [comment]
> Scheduling as a cron job in the general settings (this was the first place I found that I could add a snippet)

This is wrong, you have to execute the driver function from control panel: https://zwave-js.github.io/zwave-js-ui/#/usage/driver_function
